### PR TITLE
Fixing a small issue with chamber numbering

### PR DESCRIPTION
**Pull request by giovanni-mocellin**
gemValidation::findIndex() function faulty => eliminated, in favour of an inline simple calculation of the chamber index

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ __init__.py
 .#*
 #*#
 *~
+CondFormats/DataRecord/interface/L1TMuonEndCapParamsRcd.h
```
Comment (jshlee) [inline .gitignore:7]: dont add files here

File: CondFormats/DataRecord/src/L1TMuonEndCapParamsRcd.cc [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+// Created:
 
-// automatically generated by make_records.pl
-
-#include "CondFormats/DataRecord/interface/L1TMuonEndCapParamsRcd.h"
+#include "CondFormats/DataRecord/interface/L1TMuonEndcapParamsRcd.h"
 #include "FWCore/Framework/interface/eventsetuprecord_registration_macro.h"
 
-EVENTSETUP_RECORD_REG(L1TMuonEndCapParamsRcd);
+EVENTSETUP_RECORD_REG(L1TMuonEndcapParamsRcd);
```
Comment (jshlee) [inline CondFormats/DataRecord/src/L1TMuonEndCapParamsRcd.cc:12]: this has no changes

**Comment by jshlee** [comment]
dont add root files

**Comment by giovanni-mocellin** [comment]
Sorry! I do not know why yet, but those files were automatically fetched when I pulled the code on my laptop, and probably I was pushing them back when I asked for the PR...
Also those files are not good when trying to run...

**Comment by giovanni-mocellin** [comment]
I will make a new pull request without those files changed

**Pull request closed without merging**